README.md

```diff
@@ -53,21 +55,17 @@ The GDK and its starter projects are available on GitHub.
 **NOTE:** This link may give you a 404. 
 In order to get access to this fork, you need to link your GitHub account to a verified Epic Games account, and to have agreed to Epic's license. You will not be able to use the GDK for Unreal without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github).
 
-## SpatialOS Documentation
-* For guidance on SpatialOS concepts see the documentation on the [SpatialOS website](https://docs.improbable.io/reference/latest/shared/concepts/spatialos). <br/>
-* For information on the SpatialOS GDK for Unreal, we recommend you use the [GDK documentation on the SpatialOS website](https://docs.improbable.io/unreal/pre-alpha/index) because the documentation is designed for the website, however, you can look at the [documentation on GitHub](https://github.com/spatialos/UnrealGDK/tree/master/docs).
-
 ## Installation and setup
-For prerequisites, installation and setup, see the [Installation and setup]({{urlRoot}}/setup-and-installing.md) documentation.
+For prerequisites, installation and setup, see the [Installation and setup](https://docs.improbable.io/unreal/latest/setup-and-installing) documentation.
```

Comment:
DO you think it's worth adding:

"...see the [Installation and setup](https://docs.improbable.io/unreal/latest/setup-and-installing) on the documentation website.

Reply:
Actually - would be nice for all the docs website links on here but it doesn't matter too much. :-)